assets/images/desert.svg

```diff
@@ -0,0 +1,24 @@
+                        <path fill="#3281EE" d="M64.91 11.056s.008-5.014-.161-7.52c-.08-1.178-.542-1.905-1.22-2.35-.115-.075-.487-.258-.879-.423-.373-.158-.845-.365-1.739-.453-3.76-.37-16.992-.347-21.89-.244C33.654-.022 10.763-.1 4.826.402 3.6.506 2.355.713 1.36 1.855.333 3.035.288 4.357.241 5.528c-.073 1.874-.168 5.53-.168 5.53l64.838-.002z" transform="translate(-523 -174) translate(517 174) translate(6.701) translate(82.541 39.113)"/>
+                        <path fill="#BBD9FD" d="M61.43 5.493c-.098-.488-.431-.94-.717-1.137-.78-.538-1.958-.427-2.53.34-.577.772-.727 2.022.462 2.784 1.012.648 2.334.035 2.714-1.056.101-.293.132-.627.07-.931M55.565 5.493c-.098-.488-.432-.94-.717-1.137-.78-.538-1.958-.427-2.53.34-.577.772-.727 2.022.462 2.784 1.012.648 2.333.035 2.714-1.056.101-.293.132-.627.07-.931M49.7 5.493c-.098-.488-.432-.94-.718-1.137-.78-.538-1.957-.427-2.53.34-.577.772-.726 2.022.463 2.784 1.012.648 2.333.035 2.713-1.056.102-.293.133-.627.072-.931" transform="translate(-523 -174) translate(517 174) translate(6.701) translate(82.541 39.113)"/>
+                        <path fill="#3281EE" d="M18.1 29.003c-.165 0-.396-.044-.733-.38-.357-.356-1.173-1.582-.279-2.512.129-.134.913-.86 1.067-.995l.707-.62-.642-.686c-.285-.304-.631-.662-1.04-.997-.3-.247-.377-.612-.39-.874-.022-.475.157-.976.456-1.274.291-.292.634-.44 1.017-.44.42 0 .867.177 1.257.498.274.226.523.476.742.713l-.003.027.444.467.441.484s.73-.418 1.77-1.86c.255-.351.646-.461 1.007-.461.614 0 1.258.455 1.532 1.082.124.285.187.799-.096 1.166-.386.5-1.04 1.349-1.466 1.796l-.602.573.894.748c.19.159.37.308.536.465.472.439.601.93.391 1.462-.256.653-.95 1.145-1.615 1.145-.368 0-.714-.148-1.026-.44-.292-.27-.592-.527-.85-.745l-.603-.509-.59.522c-.258.228-.469.474-.655.692-.3.35-.81 1.143-1.671.953M40.161 29.003c-.164 0-.396-.044-.733-.38-.357-.356-1.173-1.582-.279-2.512.129-.134.913-.86 1.067-.995l.707-.62-.642-.686c-.285-.304-.631-.662-1.04-.997-.299-.247-.377-.612-.39-.874-.022-.475.156-.976.455-1.274.292-.292.634-.44 1.018-.44.42 0 .867.177 1.256.498.275.226.524.476.743.713l-.002.027.444.467.44.484s.73-.418 1.77-1.86c.255-.351.646-.461 1.007-.461.614 0 1.258.455 1.532 1.082.125.285.188.799-.096 1.166-.385.5-1.04 1.349-1.466 1.796l-.602.573.893.748c.192.159.371.308.536.465.473.439.603.93.393 1.462-.257.653-.951 1.145-1.615 1.145-.369 0-.714-.148-1.028-.44-.291-.27-.591-.527-.85-.745l-.602-.509-.59.522c-.258.228-.469.474-.655.692-.3.35-.81 1.143-1.671.953M38.472 37.882c-5.019-.528-5.018-.826-10.035-1.554-2.066-.298-2.484 2.914-.413 3.214 5.016.729 5.016 1.026 10.037 1.554 2.07.219 2.496-2.995.411-3.214" transform="translate(-523 -174) translate(517 174) translate(6.701) translate(82.541 39.113)"/>
+                    </g>
+                </g>
+            </g>
+        </g>
+    </g>
+</svg>
```

Comment:
Are SVGs optimized before being served?

Reply:
nope, we don't any tool currently to process the stack assets